src/fsharp/lexhelp.fs

```diff
@@ -335,14 +337,18 @@ module Keywords =
             match s with 
             | "__SOURCE_DIRECTORY__" ->
```

Comment:
In F# code, `__SOURCE_DIRECTORY__` and `__SOURCE_FILE__` are substituted with string literals: https://msdn.microsoft.com/en-us/library/dd233234.aspx